src/main/java/edu/hm/hafner/analysis/parser/Gcc4CompilerParser.java

```diff
@@ -23,7 +23,7 @@ public class Gcc4CompilerParser extends LookaheadParser {
     private static final long serialVersionUID = 5490211629355204910L;
 
     private static final String GCC_WARNING_PATTERN =
-            ANT_TASK + "(.+?):(\\d+):(?:(\\d+):)? ?([wW]arning|.*[Ee]rror): (.*)$";
+            "(?!\\[javac\\])" + ANT_TASK + "(.+?):(\\d+):(?:(\\d+):)? ?([wW]arning|.*[Ee]rror): (.*)$";
```

Comment:
Can you let the REGEXP unchanged and add a similar` && !javac` condition in `isLineInteresting` instead? This should be faster.